docs/data-analysis.md

```diff
@@ -0,0 +1,44 @@
+# Analyzing bot data
+
+After performing backtests, or after running the bot for some time, it will be interresting to analyze the results your bot generated.
+
+A good way for this is using Jupyter (notebook or lab) - which provides an interactive environment to analyze the data.
+
+The following helpers will help you loading the data into Pandas DataFrames, and may also give you some starting points in analyzing the results.
+
+## Backtesting
+
+To analyze your backtest results, you can [export the trades](#exporting-trades-to-file).
+You can then load the trades to perform further analysis.
+
+A good way for this is using Jupyter (notebook or lab) - which provides an interactive environment to analyze the data.
+
+Freqtrade provides an easy to load the backtest results, which is `load_backtest_data` - and takes a path to the backtest-results file.
+
+``` python
+from freqtrade.data.btanalysis import load_backtest_data
+df = load_backtest_data("user_data/backtest-result.json")
+
+# Show value-counts per pair
+df.groupby("pair")["sell_reason"].value_counts()
+
+```
+
+This will allow you to drill deeper into your backtest results, and perform analysis which would make the regular backtest-output unreadable.
```

Comment:
"analysis which would make the regular backtest-output unreadable." -- this is incorrect sentence. Analisys would not make output unreadable.

Reply:
you're right - it would make it difficult to digest since it'll be confusing with too many columns/tables.

Reply:
"which __otherwise__ would make the regular backtest-output very difficult to digest due to information overload."